example/echo.dart

```diff
@@ -2,7 +2,7 @@ library echo;
 
 import "dart:html";
 import 'package:logging/logging.dart';
-import "package:sockjs_client/sockjs.dart" as SockJS;
+import "package:sockjs_client/sockjs_client.dart" as SockJS;
```

Comment:
it was totally the wrong path .. i think the example must not have been able to work.